lib/hypothesis/world.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Hypothesis
+  module World
+    class <<self
```

Comment:
I don't know if rubocop issues an opinion here, but I'd usually write `class << self`

Reply:
rubocop doesn't seem to have an opinion either way, which is surprising as it sure seems to like having opinions.